UnitsNet.Tests/CustomCode/SpeedTests.cs

```diff
@@ -68,6 +68,12 @@ public class SpeedTests : SpeedTestsBase
 
         protected override double MillimetersPerHourInOneMeterPerSecond => 3600000;
 
+        protected override double InchesPerSecondInOneMeterPerSecond => 39.3700787;
```

Comment:
And, for the second pull request, repeat all but the first steps, this way both branches are based on origin/master and don't have any commits from the other branch.